src/wallet/wallet_tx_builder.cpp

```diff
@@ -177,25 +192,232 @@ SpendableInputs WalletTxBuilder::FindAllSpendableInputs(
     return wallet.FindSpendableInputs(selector, minDepth, std::nullopt);
 }
 
-InputSelectionResult WalletTxBuilder::ResolveInputsAndPayments(
-        const CWallet& wallet,
+static size_t NotOne(size_t n)
```

Comment:
```suggestion
static size_t PadOutputCount(size_t n)
```

Reply:
Thanks! I desperately wanted a better name for this 😆